packages/app-admin-ui/client/providers/Hooks.js

```diff
@@ -5,7 +5,10 @@ const defaultHooks = {
   // Intentionally empty for now
 };
 
-export const useUIHooks = () => useContext(HooksContext);
+export const useUIHooks = key => {
+  const hooks = useContext(HooksContext);
```

Comment:
I tried using `useList()` to get list key here but later realise that you can have hooks outside of `<ListProvider />` which will break this. if `key` here is null, you still have this working without issue.